Cards/MadGraph5_aMCatNLO/Others/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8.json

```diff
@@ -0,0 +1,27 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "3",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massZ.dat",
```

Comment:
Just to confirm, for every sample involving a Z (including ZW, we are using Z.dat)?